tests/pkg_arduino_sdi_12/Makefile

```diff
@@ -5,6 +5,7 @@ USEPKG += arduino_sdi_12
 BOARD_WHITELIST := \
   arduino-leonardo \
   arduino-mega2560 \
+  remote-revb \
```

Comment:
Is the `BOARD_WHITELIST` still justified or should this now run on any board?

Reply:
I can remove `BOARD_WHITELIST` and then see with Murdock which boards build and which don't.
But that it builds doesn't mean that it runs okay, and I only have the arduino-mega2560 and remote-revb to test.

For example in the upstream library we can read:

```
 * @note  The ESP32 and ESP8266 are fast enough processors that they can take the
 * time to read the core 'micros()' function still complete the other processing needed
 * on the serial bits.  All of the other processors using the Arduino core also have the
 * micros function, but the rest are not fast enough to waste the processor cycles to
 * use the micros function and must use the faster assembly macros to read the
 * processor timer directly.
```

https://github.com/EnviroDIY/Arduino-SDI-12/blob/master/src/SDI12_boards.h#L42

Which is the policy regarding `BOARD_WHITELIST`?
Should we let Murdock build as many as possible, or only those that are known to work?

Reply:
Building on many platforms can reveal bugs.
IMHO Whitelists / Blacklists should always be accompanied by a comment on why they are there.
In this case, adding a blacklist of boards that can't keep up with the line data rate would be the way to go IMHO.

Maybe add an empty BLACKLIST with a comment that slow boards should be added there.

Reply:
Ok done.